Lesson-6/assignment/src/components/Employee.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react'
+import { Card, Col, Row, Layout, Alert, message, Button } from 'antd';
+
+import Common from './Common';
+
+class Employer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    const { payroll } = this.props;
+    const updateInfo = (error, result) => {
+      if (!error) {
+        this.checkEmployee();
+      } else {
+        console.log(error);
+      }
+    }
+
+    this.getPaidEvent = payroll.GetPaid(updateInfo);
```

Comment:
good job! It's also worth to think about is why `this.newFundEvent = payroll.NewFund(updateInfo);` is not needed here but needed in common.js